api/EffectTiming.json

```diff
@@ -35,11 +51,19 @@
             "version_added": null
           },
           "samsunginternet_android": {
+            "alternative_name": "AnimationEffectTimingProperties",
             "version_added": true
           },
-          "webview_android": {
-            "version_added": true
-          }
+          "webview_android": [
+            {
+              "version_removed": "67"
+            },
+            {
+              "version_added": "52",
+              "version_removed": "66",
+              "alternative_name": "AnimationEffectTimingProperties"
+            }
+          ],
         },
```

Comment:
The build fails because your JSON is malformed. I believe you need to remove this line:

```suggestion
```

Tip: You can always test locally using `npm test`

Reply:
Shut up phone broke and mind me do wat you get paid to do that keep me up from zzzz Nd fix wat I jack up due stress Nd no good phones 😵‼️📱